ambry-router/src/main/java/com.github.ambry.router/GetBlobOperation.java

```diff
@@ -70,8 +70,8 @@
  */
 class GetBlobOperation extends GetOperation {
   // the callback to use to complete the operation.
-  private final OperationCompleteCallback operationCompleteCallback;
-  // whether the operationCompleteCallback has been called already.
+  private final OperationCallback operationCallback;
+  // whether the operationCallback has been called already.
   private final AtomicBoolean operationCallbackInvoked = new AtomicBoolean(false);
```

Comment:
The `operationCallback` that this refers to is no longer part of `operationCallback`, so this variable name is a bit confusing. Perhaps calling it `callbackInvoked` or `getOperationCallbackInvoked` would be better.